.gitignore

```diff
@@ -99,3 +99,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# visual studio code
+.vscode
```

Comment:
Prefer global gitignore for this "IDE relative" files.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Or the `./git/info/exclude` file in your project folder.